Emby.Server.Implementations/LiveTv/TunerHosts/M3uParser.cs

```diff
@@ -61,7 +61,7 @@ public Task<Stream> GetListingsStream(string url, CancellationToken cancellation
                     UserAgent = _appHost.ApplicationUserAgent
                 });
             }
-            return Task.FromResult(_fileSystem.OpenRead(url));
+            return Task.FromResult((Stream)File.OpenRead(url));
```

Comment:
Is this cast needed?

Reply:
Yes
FileStream -> Stream does not happen automatically. Maybe BaseStream is possible as well.